app/models/world_location_news.rb

```diff
@@ -17,4 +17,20 @@ def title
   def description
     @content_item.content_item_data["description"]
   end
+
+  def ordered_featured_documents
+    @content_item.content_item_data.dig("details", "ordered_featured_documents")&.map do |document|
```

Comment:
Here, I don't think we technically need the '&' since I made ordered_featured_documents mandatory on the schema. Not necessarily anti keeping this in though